src/olympia/amo/management/commands/seed_data.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Seed the _init data dir with fresh data from the database'
```

Comment:
This help text really only makes sense if you know that `settings.DATA_BACKUP_INIT` defaults to `_init` and that's the name of the directory in the backups directory... at which point you don't need the help text 🙃

```suggestion
    help = 'Reset and seed the database with initial data, generated add-ons, and data from AMO prod'
```

(or something)

Reply:
Fixed